### ISSUE TITLE
type="text" gets stripped from form input

### ISSUE THREAD

**Issue by kevinGodell** [jantimon/html-webpack-plugin#1396]
## Expected behaviour

`type="text"` should not be stripped from form `input` element in html output.

## Current behaviour

`type="text"` is removed from form input element.

## Environment

Trouble started after upgrading from 3.2.0 to 4.0.4.

Tell us which operating system you are using, as well as which versions of Node.js, npm, webpack, and html-webpack-plugin. Run the following to get it quickly:

```
Node.js v12.11.0
darwin 18.5.0
npm 6.14.2
webpack@4.42.1 
html-webpack-plugin@4.0.4
```

### Config

Copy the minimal `webpack.config.js` to produce this issue:

```js
const HtmlWebpackPlugin = require('html-webpack-plugin');
const { resolve } = require('path');

module.exports = {
    mode: 'production',
    entry: {
        main: resolve(__dirname, './src/app.js')
    },
    output: {
        path: resolve(__dirname, './dist')
    },
    plugins: [
        new HtmlWebpackPlugin({
            template: resolve(__dirname, './src/index.html')
        })
    ]
};
```

Copy your template file if it is part of this issue:

```html
<!DOCTYPE html>
<html>
<head>
  <title>My App</title>
</head>
<body>
<form>
  <input type="text" name="myText" id="myText"/>
  <input type="number" name="myNumber" id="myNumber"/>
  <input type="submit"/>
</form>
</body>
</html>
```

## Relevant Links
minimal reproduction
[html-webpack-plugin-bug](https://github.com/kevinGodell/html-webpack-plugin-bug)

**Comment by jantimon** [comment]
```
type="text" should not be stripped from form input element in html output.
``` 

`type="text"` is the default and is removed during html minifcation by `html-minifier-terser`.

if you don' want to enable minifcations please set `minify: false`

**Comment by kevinGodell** [comment]
Ah, I thought it was a bug introduced with version 4 since it did not do this with version 3. I only noticed because I had a css selector input[type=text] that started looking un-styled after upgrading to v4. After looking more closely at your docs, I see I can pass an object to minify to further customize the html minification to keep the "redundant properties" and still take advantage of the other features. I updated my test repo to reflect the customized settings. https://github.com/kevinGodell/html-webpack-plugin-bug/commit/a98ac1bdd2f54f66101b4f593fec7a853302f2aa#diff-11e9f7f953edc64ba14b0cc350ae7b9dR13

Thank you for taking the time to respond.
